packages/host/app/components/operator-mode/stack-item.gts

```diff
@@ -476,6 +517,11 @@ export default class OperatorModeStackItem extends Component<Signature> {
         --icon-bg: var(--boxel-dark);
       }
 
+      .header-icon {
+        width: var(--boxel-header-icon-width, 20px);
+        height: var(--boxel-header-icon-height, 20px);
```

Comment:
@jurgenwerk do we have boxel css vars that we can use instead of hard coded pixels?

Reply:
I think we don't need those hardcoded pixels, addressed in https://github.com/cardstack/boxel/pull/491/commits/9718a85783d0cd28cda2bb618204dafefc5b1d35